### PR TITLE
OCP4: replies to AU-3, 3(1)

### PR DESCRIPTION
**Pull request by jhrozek**
OCP4: AU-3(1): custom audit policies are not supported, but the
WriteRequestBodies should satisfy the control

We'll follow up with https://issues.redhat.com/browse/CMP-974

OCP4: AU-3 is inherently met

All the required events are audited by default with all the profiles.

### CODE REVIEW

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 1]
```diff
@@ -88,22 +88,41 @@
...
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        Red Hat OpenShift supports three different audit policies which are
+        documented at:
+        https://docs.openshift.com/container-platform/4.7/security/audit-log-policy-config.html
 
-        https://issues.redhat.com/browse/CMP-157
+        At the moment, custom audit policies are not supported. However,
+        the WriteRequestBodies and AllRequestBodies policies should satisfy
+        all the information the control asks for
```
Comment (JAORMX) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml:125]: Do we want to keep this as planned? I don't think there's really a desire to support custom audit profiles.

Comment (jhrozek) [reply openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml:133]: Good point, new PR pushed. We do have a follow-up ticket..

Comment (JAORMX) [reply openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml:133]: Implementation status is still `planned`

**Comment by jhrozek** [comment]
sorry, forgot to --amend the commit earlier. Now they should be OK

**Pull request merged**